book_class.rb

```diff
@@ -1,3 +1,5 @@
+require 'JSON'
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, I'm getting the following error:

![image](https://github.com/mahammad-mostafa/school-library/assets/77069011/e1144093-5ac1-4665-a6ef-29b5a20fbcca)

Once I changed it to lowercase `json` it worked fine! 🤓 🔧